snippets/markdown-basic-readme/README.md

```diff
@@ -0,0 +1,34 @@
+# Simple README Markdown Snippet
+
+This snippet generates boilerplate code for creating a simple README document using Markdown.
+
+<br/>
+
+## Prerequisites
+
+This snippet's dependencies are an up-to-date version of the VSCode IDE:
+* Install VSCode: https://code.visualstudio.com
+
+<br/>
+
+## Usage
+
+Using the snippet requires copying the contents of the snippet.nson file and pasting them into either the `markdown.json` snippet configuration file, or the global snippets configuration file in VSCode. The user snippets configuration files are accessible via VScode `Preferences` under the `Configure User Snippets` option.
```

Comment:
```suggestion
Using the snippet requires copying the contents of the snippet.json file and pasting them into either the `markdown.json` snippet configuration file, or the global snippets configuration file in VSCode. The user snippets configuration files are accessible via VScode `Preferences` under the `Configure User Snippets` option.
```

Assuming this was supposed to be `json` and not some other object notation format named `nson`

Reply:
Correct assumption...  :smile: